verilog/analysis/checkers/module_filename_rule.h

```diff
@@ -31,26 +31,17 @@ namespace analysis {
 class ModuleFilenameRule : public verible::TextStructureLintRule {
  public:
   using rule_type = verible::TextStructureLintRule;
-  static absl::string_view Name();
 
-  // Returns the description of the rule implemented formatted for either the
-  // helper flag or markdown depending on the parameter type.
-  static std::string GetDescription(DescriptionType);
+  static const LintRuleDescriptor &GetDescriptor();
 
   ModuleFilenameRule() {}
 
   absl::Status Configure(absl::string_view configuration) override;
-  void Lint(const verible::TextStructureView&, absl::string_view) override;
+  void Lint(const verible::TextStructureView &, absl::string_view) override;
```

Comment:
Again, we hug & left, I believe.

Reply:
Fixed (clang tidy was picking up on my tendency to right hug...)